include/low_pass_filter.h

```diff
@@ -2,7 +2,20 @@
 #define ROCKETLIB_LOW_PASS_FILTER_H
 
 #include "common.h"
+#include <stdbool.h>
+#include <stdint.h>
 
-w_status_t low_pass_filter_init(void);
+// Macro to calculate the sample frequency based on the time difference in milliseconds
+#define SAMPLE_FREQ(time_diff_ms) (1000.0 / (time_diff_ms))
```

Comment:
Replace with function

Reply:
@taisirhassan https://docs.waterloorocketry.com/general/standards/embedded-coding-standard.html#embedded-coding-standard for reference